demo/laneLoader.js

```diff
@@ -152,6 +152,7 @@ async function parseLanes(bytesArray, FlatbufferModule, supplierNum, annotationM
 }
 
 
+// TODO we are not using this function and will need to rewrite is to accomodate annotations and amomalies. Recmommend removing
```

Comment:
accomodate annotations and anomalies? Could you explain what you mean?

Reply:
This function was probably written with the intent of refactoring that big `createLaneGeometriesOld` function that I added my anomaly stuff too. That function also has annotations built in too. If we want to refactor, it would be easier to rewrite than to use the old stuff that doesn't work with annotations (markers in the lanes that can be moved) and anomalies.